data/presets/natural/stone.json

```diff
@@ -13,7 +18,13 @@
     "terms": [
         "boulder",
         "stone",
-        "rock"
+        "rock",
+        "findling",
+        "glacial erratic",
```

Comment:
the preset's name shouldn't be repeated in the list of `terms`

Reply:
I'm a little unclear on how aliases work - so if boulder is an alias, it would still operate the same way as if it were in terms without duplicating it?

Reply:
Yes, exactly. Basically the set of `name + aliases + terms` defines all the possible "words" a preset can be found when searching for it. Whether an individual "word" is contained in the `name` or `aliases` or `terms` only changes how the preset is displayed in the UI (`name`) and influences the order of search results (presets that only match a word from the list of `terms` are sorted below matching `name` or `aliases`).